shared-data/contrib/autoajax/autoajax.js

```diff
@@ -133,15 +133,6 @@ prepare_new_content = function(selector) {
             });
         }
     });
-    var $form = $(selector).find('form#form-search');
```

Comment:
autoajax used to handle all search queries to the form. Now those are processed again by the [code in the topbar.js](https://github.com/mailpile/Mailpile/pull/1815/files#diff-146d89056853750bae4bf5d688a842c3R35).

That code will forward all non special-queries back to autoajax.